WordPress/Classes/ViewRelated/Comments/EditReplyViewController.m

```diff
@@ -11,6 +40,122 @@ - (void)viewDidLoad
+                                                              relatedBy:NSLayoutRelationEqual
+                                                                 toItem:self.textView
+                                                              attribute:NSLayoutAttributeLeft
+                                                             multiplier:1
+                                                               constant:0]];
+        
+        [self.view addConstraint:[NSLayoutConstraint constraintWithItem:self.suggestionsTableView
+                                                              attribute:NSLayoutAttributeRight
+                                                              relatedBy:NSLayoutRelationEqual
+                                                                 toItem:self.textView
+                                                              attribute:NSLayoutAttributeRight
+                                                             multiplier:1
+                                                               constant:0]];
+        
+        // Pin the suggestions view top to the super view top, less a 30 pt margin
+        [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|-30.0-[suggestionsview]"
+                                                                          options:0
+                                                                          metrics:nil
+                                                                            views:views]];
+        
+        // Pin the suggestions view bottom to the bottom of the reply box
+        [self.view addConstraint:[NSLayoutConstraint constraintWithItem:self.suggestionsTableView
+                                                              attribute:NSLayoutAttributeBottom
+                                                              relatedBy:NSLayoutRelationEqual
+                                                                 toItem:self.textView
+                                                              attribute:NSLayoutAttributeBottom
+                                                             multiplier:1
+                                                               constant:0]];
+        
+        // listen to textView changes
+        self.textView.delegate = self;
+    }
+}
+
+#pragma mark - UITextViewDelegate
+
+- (BOOL)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range replacementText:(NSString *)text
+{
+    // Get all the text
+    NSString *textViewText = textView.text;
+    NSRange textViewRange = NSMakeRange(0, range.location);
+    NSString *pretext = [[textViewText substringWithRange:textViewRange] stringByAppendingString:text];
+    NSArray *words = [pretext componentsSeparatedByCharactersInSet:[NSCharacterSet whitespaceAndNewlineCharacterSet]];
+    [self.suggestionsTableView showSuggestionsForWord:[words lastObject]];
```

Comment:
It is, but in the case of EditReplyViewController, it's not a ReplyTextView, it is an iOS7CorrectedTextView (child of UITextView), unless I'm missing something.  That and we wanted to eventually remove it from ReplyTextView too, to remove the need for the ReplyTextView to have that delegate.

So, no changes here - is that OK?